src/Tooltip.js

```diff
@@ -27,7 +20,232 @@ function TooltipBase({direction, children, className, text, noDelay, align, wrap
   )
 }
 
-const Tooltip = styled(TooltipBase)(COMMON)
+const Tooltip = styled(TooltipBase)`
+  position: relative;
```

Comment:
R.I.P. so much CSS for this component 😭 @mxstbr Is it totally bad to put this in a separate js file and import it?